tools/policy-gen/protoc-gen-kumapolicy/crd.go

```diff
@@ -0,0 +1,221 @@
+package main
+
+import (
+	"bytes"
+	"go/format"
+	"html/template"
+
+	"github.com/pkg/errors"
+	"google.golang.org/protobuf/compiler/protogen"
+
+	"github.com/kumahq/kuma/tools/resource-gen/genutils"
+)
+
+// CustomResourceTemplate for creating a Kubernetes CRD to wrap a Kuma resource.
+var CustomResourceTemplate = template.Must(template.New("custom-resource").Parse(`
```

Comment:
Dn't we already generate CRDs from Kuma resources? @parkanzky is that what you meant?

Reply:
Yes, we do in `tools/resource-gen/main.go`, but it doesn't make sense to reuse the template for 2 reasons:
* templates are different, because `Spec` fields are different
* we want to get rid of `tools/resource-gen/main.go` in the future because instead of accepting proto as an input, it imports all protos and generates resources for everything at once. It won't work well if you have `proto` files inside plugins directories